src/openrct2-ui/windows/Finances.cpp

```diff
@@ -83,55 +81,63 @@ enum
     WIDX_SCENERY_AND_THEMING,
 };
 
-#pragma region Widgets
+#pragma region Measurements
+
+static constexpr const int32_t WH_SUMMARY = 309;
+static constexpr const int32_t WH_RESEARCH = 207;
+static constexpr const int32_t WH_OTHER_TABS = 257;
+static constexpr const int32_t WW_RESEARCH = 320;
+static constexpr const int32_t WW_OTHER_TABS = 530;
+static constexpr const int32_t RSH_SUMMARY = 266;
+static constexpr const int32_t RSH_RESEARCH = 164;
+static constexpr const int32_t RSH_OTHER_TABS = 214;
+static constexpr const int32_t RSW_RESEARCH = WW_RESEARCH;
+static constexpr const int32_t RSW_OTHER_TABS = WW_OTHER_TABS;
 
-#define TAB_WIDGETS \
+#pragma endregion
+
+#pragma region Widgets 
+
+#define MAIN_FINANCES_WIDGETS(TITLE, RSW, RSH, WW, WH) \
+    WINDOW_SHIM(TITLE, WW, WH), \
+    MakeWidget({0, 43}, {RSW, RSH}, WindowWidgetType::Resize, WindowColour::Secondary), \
     MakeTab({  3, 17}, STR_FINANCES_SHOW_SUMMARY_TAB_TIP      ), \
     MakeTab({ 34, 17}, STR_FINANCES_SHOW_CASH_TAB_TIP         ), \
     MakeTab({ 65, 17}, STR_FINANCES_SHOW_PARK_VALUE_TAB_TIP   ), \
     MakeTab({ 96, 17}, STR_FINANCES_SHOW_WEEKLY_PROFIT_TAB_TIP), \
     MakeTab({127, 17}, STR_FINANCES_SHOW_MARKETING_TAB_TIP    ), \
     MakeTab({158, 17}, STR_FINANCES_RESEARCH_TIP              )
 
-static rct_widget _windowFinancesSummaryWidgets[] =
+
+static rct_widget window_finances_summary_widgets[] =
 {
-    WINDOW_SHIM(STR_FINANCIAL_SUMMARY, WW_OTHER_TABS, WH_SUMMARY),
-    MakeWidget        ({  0,  43}, {530, 266}, WindowWidgetType::Resize,  WindowColour::Secondary),
-    TAB_WIDGETS,
+    MAIN_FINANCES_WIDGETS(STR_FINANCIAL_SUMMARY, RSW_OTHER_TABS, RSH_SUMMARY, WW_OTHER_TABS, WH_SUMMARY),
     MakeWidget        ({130,  50}, {391, 211}, WindowWidgetType::Scroll,  WindowColour::Secondary, SCROLL_HORIZONTAL              ),
     MakeSpinnerWidgets({ 64, 279}, { 97,  14}, WindowWidgetType::Spinner, WindowColour::Secondary, STR_FINANCES_SUMMARY_LOAN_VALUE), // NB: 3 widgets.
     WIDGETS_END,
 };
 
-static rct_widget _windowFinancesCashWidgets[] =
+static rct_widget window_finances_cash_widgets[] =
```

Comment:
stick to the old naming it was the most correct following the coding style. This comment goes for all of the static local globals.